packages/stream-model-instance-handler/src/__tests__/model-instance-document-handler.test.ts

```diff
@@ -329,13 +329,7 @@ describe('ModelInstanceDocumentHandler', () => {
     expect(serialized).toEqual(signed)
   })
 
-  it('throws error for deterministic genesis commit with data', async () => {
-    await expect(
-      ModelInstanceDocument.makeGenesis(context.api, COMMITS.genesis.data, { deterministic: true })
-    ).rejects.toThrow(/Initial content must be null/)
-  })
-
-  it('Takes controller from authenticated DID if controller not specified', async () => {
+  it('Does not sign commit if no content', async () => {
```

Comment:
now two tests with the same name?  See the next test on line 337.  Is this supposed to be `Takes controller from authenticated DID if controller not specified`?